apps/api/src/app/workflows-v2/usecases/post-process-workflow-update/post-process-workflow-update.usecase.ts

```diff
@@ -155,18 +156,22 @@ export class PostProcessWorkflowUpdate {
 
   private updateIssuesOnWorkflow(
     workflow: WorkflowInternalResponseDto,
-    workflowIssues: Record<keyof WorkflowResponseDto, RuntimeIssue[]>,
-    stepIssuesMap: Record<string, StepIssues>
+    stepIssuesMap?: Record<string, StepIssues>,
```

Comment:
This is an attempt to align workflow and step issue type definition as part of this function

```suggestion
    stepIssues?: Record<keyof StepResponseDto, StepIssues>,
```

Reply:
you missunderstood the parameter, the string is the stepId

Reply:
This is one of the reasons it is hard for me to work with Record<string, Foo> there is no good way to understand the object by the type.